docs/src/docs/asciidoc/graalvm-setup.adoc

```diff
@@ -0,0 +1,51 @@
+= Setting up GraalVM with native-image support
+
+image:https://www.graalvm.org/resources/img/logo-colored.svg[GraalVM]
+
+Working GraalVM distribution with `native-image` installable and `GRAALVM_HOME` and/or `JAVA_HOME` environment variables set, is prequisite for successful *native-image* building.
+
+Following are the steps needed to obtain and setup GraalVM environment.
+
+> Alternatively, we have provided a [script](../scripts/downloadGraalVM.sh) for downloading and setting up latest nightly in CI environment.
```

Comment:
This [doesn't render properly](https://graalvm.github.io/native-build-tools/0.9.2/graalvm-setup.html).

Reply:
Good catch, fixed!